ci-operator/step-registry/ipi/conf/aws/oidc-creds/ipi-conf-aws-oidc-creds-commands.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+MPREFIX="${SHARED_DIR}/manifest"
+TPREFIX="${SHARED_DIR}/tls"
+WEB_IDENTITY_TOKEN_FILE="/var/run/secrets/openshift/serviceaccount/token"
+
+
+cp /var/run/secret/bound-sa-signing-key/service-account.key "${TPREFIX}_bound-service-account-signing-key.key"
+
+cat >> "${MPREFIX}_cluster-authentication-02-config.yaml" << EOF
+apiVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+  name: cluster
+spec:
+  serviceAccountIssuer: https://do-not-remove-shared-oidc-oidc-discovery.s3.amazonaws.com
+
+EOF
+
+CONFIG="[default]
+arn:aws:iam::460538899914:role/do-not-remove-shared-oidc-cred-minter-s3-role
+web_identity_token_file = ${WEB_IDENTITY_TOKEN_FILE}
+"
+BASE64CONFIG=$(echo "${CONFIG}" | base64 -w0)
+cat >> "${MPREFIX}_secret-credentials-cred-minter-s3.yaml" << EOF
+apiVersion: v1
+data:
+  credentials: ${BASE64CONFIG}
+kind: Secret
+metadata:
+  name: cloud-credential-operator-s3-creds
+  namespace: openshift-cloud-credential-operator
+
+EOF
```

Comment:
Ever since openshift/cloud-credential-operator#276 and openshift/cloud-credential-operator#278 , these creds are no longer needed.

Reply:
@joelddiaz dropped it.